public/app/features/dashboard-scene/edit-pane/VizPanelEditableElement.tsx

```diff
@@ -69,50 +87,72 @@ export class VizPanelEditableElement implements EditableDashboardElement, BulkAc
       return undefined;
     }, [layoutElement]);
 
-    const { options, fieldConfig, _pluginInstanceState } = panel.useState();
-    const dataProvider = sceneGraph.getData(panel);
-    const { data } = dataProvider.useState();
-
-    const visualizationOptions = useMemo(() => {
-      const plugin = panel.getPlugin();
-      if (!plugin) {
-        return [];
-      }
-
-      return getVisualizationOptions2({
-        panel,
-        data,
-        plugin: plugin,
-        eventBus: panel.getPanelContext().eventBus,
-        instanceState: _pluginInstanceState,
-      });
-      // eslint-disable-next-line react-hooks/exhaustive-deps
-    }, [data, panel, options, fieldConfig, _pluginInstanceState]);
-
     const categories = [panelOptions];
     if (layoutCategory) {
       categories.push(layoutCategory);
     }
 
-    categories.push(...visualizationOptions);
-
     return categories;
   }
 
   public onDelete = () => {
     const layout = dashboardSceneGraph.getLayoutManagerFor(this.panel);
     layout.removePanel?.(this.panel);
   };
-
-  public renderActions(): ReactNode {
-    return (
-      <>
-        <Button size="sm" variant="secondary">
-          <Trans i18nKey="panel.header-menu.edit">Edit</Trans>
-        </Button>
-        <Button size="sm" variant="secondary" icon="copy" />
-        <Button size="sm" variant="destructive" fill="outline" onClick={this.onDelete} icon="trash-alt" />
-      </>
-    );
-  }
 }
+
+const getStyles = (theme: GrafanaTheme2) => ({
```

Comment:
It'd be a lil easier to follow the code if you move the `getStyles` after the component.